faq/faq.rst

```diff
@@ -180,7 +180,10 @@ As the error message suggests, you can (cautiously) increase this limit by alter
 Can I run Citus on Heroku or Amazon RDS?
 ----------------------------------------
 
-At this time Heroku and Amazon do not support running Citus directly on top of Heroku PostgreSQL or Amazon RDS. It is up to them if/when they enable the Citus extension. If you are looking for something similar, `Citus Cloud <https://www.citusdata.com/product/cloud>`_ is our database-as-a-service which we fully manage for you. It runs on top of AWS (like both RDS and Heroku PostgreSQL) and should provide a very similar product experience, with the addition of Citus' horizontal scaling.
+Yes, Citus integrates with both of these platforms.
+
+* The `Citus add-on <https://elements.heroku.com/addons/citus>`_ provisions a Citus cluster for use by a Heroku app.
```

Comment:
We may want to split this into 2 or 3 FAQ's now. We do support Heroku via the addon. We do support using your own AWS account to use/pay-for Citus Cloud.  However, we don't support RDS. The AWS Marketplace is different, in that they can basically use/provision Citus using their existing AWS account rather than create a Citus Cloud Account. However RDS is Amazon's relational database, and we are not a sanctioned extension on Postgres RDS.

Reply:
> we don't support RDS

Ah totally. I actually didn't notice the word RDS in the faq entry, just saw amazon. 😬 Sorry about that.